sap/cli/gcts.py

```diff
@@ -248,12 +262,13 @@ def delete(connection, args):
     """
 
     try:
-        sap.rest.gcts.simple.delete(connection, args.package)
+        repo = get_repository(connection, args.package)
+        sap.rest.gcts.simple.delete(connection, repo=repo)
```

Comment:
I am not sure we need to use the simplified API here. Just use call repo.delete directly.